### ISSUE TITLE
Disabled search indexing. 

### ISSUE THREAD

**Issue by doomsdaymatrix** [W4RH4WK/Debloat-Windows-10#259]
Running scripts left search indexing disabled and completely removed Windows Search from services.msc. Any way to restore functionality of windows search?

**Comment by W4RH4WK** [comment]
Is this entry really gone from the Services control panel?

![image](https://user-images.githubusercontent.com/328798/89074667-0535b500-d37d-11ea-868d-a924bb822c63.png)

**Comment by doomsdaymatrix** [comment]
Yes. Its just not there anymore.

**Comment by W4RH4WK** [comment]
Okay, right now I don't see any obvious reason why that service is missing. My guess would be that either some of the apps that get removed also remove the search component as it depends on that app. Or that one of the privacy settings conflicts with search, yet I wonder why it leads to the service being no longer present in the control panel.

I assume you did not run the *experimental unfuckery* script.

I'll mark this issue with help wanted, maybe something comes up. If I find the time, I'll look into it, but I am rather busy at the moment.

**Comment by doomsdaymatrix** [comment]
No, I have not run the experimental unfuckery script yet. Search indexing was also disabled and searching across all of windows is now impossible. 

Appreciate it.

**Comment by gxgl** [comment]
Hi,

Before starting, please check if the C:\Windows\system32\SearchIndexer.exe exist.

Second  try to go to Settings -> Update & Security -> Troubleshoot -> Additional Troubleshooters -> Search and indexing?
If that will not help, maybe one of the steps here https://www.windowscentral.com/how-fix-taskbar-search-not-working-windows-10 will do.

The WSearch (Windows Search) service depends on BrokerInfrastructure (Background Tasks Infrastructure Service) but that is not in the files as I can see and also the WSearch is commented in the disable-services.ps1 file.

Regards,
George

**Comment by W4RH4WK** [comment]
@doomsdaymatrix btw, as a temporary workaround I can recommend [Everything](https://www.voidtools.com/) as a third-party search indexer. I use it personally in my everyday workflow.

**Comment by doomsdaymatrix** [comment]
Highly appreciate it, @W4RH4WK 

@gxgl No, SearchIndexer.exe does not exist. Troubleshooting doesn't do anything as its unable to start windows search service. Unsurprising since it doesn't seem to exist.

**Comment by gxgl** [comment]
Hi,

I assume that you are running on the latest version of windows10 but to be sure, can you please open a cmd or powershell window, type winver, then hit enter and let me know about the version of your os?

**Comment by gxgl** [comment]
I have searched all the files in this repo and none is presenting "the feature" to delete that file, so the issue must come from another "thing", maybe an unfinished/failed update, or some antivirus, or some other third party tool action.

Anyway, let's do try to a trick here (which I hope it will work):

My system is running on Windows10 20.04 and the after searching for SearchIndexer.exe, the used file is 914Kb in size (there are more) also there is a same size file in "C:\Windows\WinSxS\amd64_windowssearchengine_31bf3856ad364e35_7.0.19041.329_none_8b9cc8000cc1dec0".
If you have it, you may copy the SearchIndexer.exe file from there to the C:\Windows\system32

-
